README.md

```diff
@@ -62,6 +62,7 @@ See the [Quickstart](https://salesforce.github.io/refocus/docs/01-quickstart.htm
         the threshold of what we should consider "unused". Set the value
         to a negative time offset like `-30d` if you want to consider a
         token unused if it has not been used in the last 30 days.
+1. If you want Refocus Collectors collecting only from sources using HTTPS, set environment variable `COLLECTOR_REQUIRE_SSL_TO_REMOTE_DATA_SOURCE` to `true`.
```

Comment:
If you want to require SSL for connections from Refocus Collectors to the remote data sources, set environment variable `COLLECTOR_REQUIRE_SSL_TO_REMOTE_DATA_SOURCE` to `true`.